docs/versioning-policy.md

```diff
@@ -0,0 +1,81 @@
+You must **not** rely on these numbers to derive any information about breaking changes or ease of updating.
+**This is not SemVer!**
+
+However, we do care a lot about providing you with all relevant information about breaking changes!
+Every release mentions all potentially breaking changes in its changelog.
+Additionally, there is a release overview with breaking changes [here](./releases.md).
+
+<br>
+<br>
+
+---
+
+## Breaking and non-breaking changes
+
+Our definition of these terms for Tobira. Defines what we guarantee and what not.
+
+### Breaking changes/stability guarantees
+
+We consider the following changes as *breaking*, highlighting them in changelogs and tracking them in [the release overview](./releases.md).
+The short tags at the beginning are the table column headers in said release overview.
+
+- *config*: Changing the configuration file in a backwards-incompatible way.
+- *auth*: Changing the auth-system in a backwards-incompatible way.
+- *cli*: Changing the stabilized parts of the CLI in a backwards-incompatible way.
+- *meili*: Requiring a new version of MeiliSearch.
+- *pg*: Dropping support for a non-EOL version of PostgreSQL.
+- *oc-version*: Requiring a new Opencast version (often by requiring a newer version of the Tobira module).
+- *oc-requirement*: Adding new Opencast requirements (e.g. having static-file authorization disabled).
+
+
+### Non-breaking changes
+
+All changes not listed in the previous section are considered *non-breaking* by us.
+We may release these changes at any time.
+And while most will likely be mentioned in the changelog somewhere, some may not.
+
+To name a few things explicitly (this is not a complete list!):
+
+- Any changes to the GraphQL API. **This API is internal and we will break it very regularly.**
+- Any changes to emitted HTML, DOM nodes, CSS, and JS.
+- Any user-facing changes, like adding features, removing features, changing features, change the design, ...
+- Database schema changes: we always provide scripts to easily migrate the database.
+  Tobira automatically runs the appropriate scripts when starting, so you don't have to do anything.
+- Anything that requires a search index rebuild.
+  These rebuilds will be performed automatically after updating Tobira and should be done within a few seconds (for tens of thousands of events/series).
+
+
+<br>
+<br>
+
+---
+
+## Stable components of Tobira
+
+See the section "Breaking and non-breaking changes" for context.
+
+### CLI (command line interface)
+
+- `tobira serve`: starts the web server.
+- `tobira worker`: runs forever syncing with Opencast and keeping the search index up to date.
+- `tobira check`: checks for some potential problems with Tobira's configuration and environment.
+  Will return a non-zero error code if some problem was found.
+- `tobira sync run [--daemon]`
+- `tobira search-index update [--daemon]`
+
+And the `-c config-file` option for all these commands.
```

Comment:
This is not a complete sentence. And while it's not the first example of that in these docs, it's certainly the most noticeable. :thinking:

Why is this not just part of the list above, anyway? That and removing the "And" would already improve this situation, I think. x)